lib/crawly/worker.ex

```diff
@@ -9,6 +9,7 @@ defmodule Crawly.Worker do
 
   # define the default worker fetch interval.
   @default_backoff 10_000
+  @start_timeout 1000
```

Comment:
I had to tweak this value a bit, as workers were stuck a bit during the startup (as requests storage was empty), so in general, it caused delays due to increasing backoff.

Another aspect - it might be wise to start workers slowly (so we're not hitting the website at the same time), which probably can be achieved if we will add some randomness here.

Reply:
perhaps a queue for worker requests?

Reply:
It would change the worker's logic quite a bit. I would return to this question separately.